app/src/components/Exam.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div class="exams text-center" >
+        <p class="exam-text common fade-in" v-if="isExam">
+            <i class="fas fa-exclamation-triangle"></i> Πραγματοποιείται εξέταση!
+        </p>
+    </div>
+</template>
+
+<script>
+    import {EventBus} from "./EventBus";
+
+    export default {
+        name: "Exam",
+
+        data() {
+            return {
+                isExam: ''
+            }
+        },
+
+        created() {
+            this.registerExamListener();
+        },
+
+        methods: {
+            registerExamListener: function () {
+               EventBus.$on('exam', exam => {
+                   this.isExam = exam;
+               });
+            }
+        }
+    }
+</script>
+
+<style lang="scss">
+    @import "../css/Exam.scss";
```

Comment:
Maybe let's use a `/scss/` folder, since we are dealing with SCSS>

Reply:
Whyyyy? Done...